claim-check/code-samples/sample-3/ClientConsumer3/ClientConsumer3.csproj

```
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+		<PropertyGroup>
+				<OutputType>Exe</OutputType>
+				<TargetFramework>net8.0</TargetFramework>
+				<ImplicitUsings>enable</ImplicitUsings>
+				<Nullable>enable</Nullable>
+				<RootNamespace>Pnp.Samples.ClaimCheckPattern</RootNamespace>
+				<LangVersion>latest</LangVersion>
+		</PropertyGroup>
+
+		<ItemGroup>
+				<Content Include="appsettings.json">
+						<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+				</Content>
+		</ItemGroup>
+		<ItemGroup>
+				<None Update="appsettings.json.template">
+						<CopyToOutputDirectory>Never</CopyToOutputDirectory>
+				</None>
+		</ItemGroup>
+
+		<ItemGroup>
+				<PackageReference Include="Azure.Identity" Version="1.11.0" />
```

Just for consistency.

```suggestion
				<PackageReference Include="Azure.Identity" Version="1.11.1" />
```